setup.py

```diff
@@ -1,9 +1,10 @@
 from __future__ import annotations
 
-import os
+import re
 import subprocess
+from pathlib import Path
 
-from setuptools import setup
+import setuptools
 
 # Version number is determined either by git revision (which takes precendence)
 # or a static version number which is updated by bump2version
```

Comment:
should this be the other way round? Static version number takes precedence, and only if it's a dev version should we even attempt to get a git version number?

Reply:
I think you are right here - I was attempting to handle an edge case, but by complicating and introducing more possibility for error in all other cases. Will rework this.